src/instruments/src/EFB/tailwind.config.js

```diff
@@ -3,7 +3,7 @@
 const reactComponentsClasses = require('../../../../node_modules/@flybywiresim/react-components/build/usedCSSClasses.json');
 
 module.exports = {
-    mode: 'jit',
+    // mode: 'jit',
```

Comment:
**question**: Is this change intentional?

Reply:
No